pkg/cmd/dockerregistry/dockerregistry.go

```diff
@@ -98,6 +98,38 @@ func Execute(configFile io.Reader) {
 		pruneAccessRecords,
 	)
 
+	// openshiftRouter groups the OpenShift related registry extensions
+	openshiftRouter := app.NewRoute().PathPrefix("/openshift/").Subrouter()
+
+	signatureRouter := openshiftRouter.NewRoute().PathPrefix("/signature/").Subrouter()
```

Comment:
@legionus I think this functionality is not related to `/v2` or `/v1' (it should work for both since it  is a custom endpoint?)